packages/transport/package.json

```diff
@@ -52,6 +52,8 @@
         "prepublish": "yarn tsx ../../scripts/prepublish.js"
     },
     "devDependencies": {
+        "@babel/preset-env": "^7.23.9",
+        "@babel/preset-typescript": "^7.23.3",
```

Comment:
Aren't those dependencies used in jest.config? https://github.com/trezor/trezor-suite/blob/develop/packages/transport/e2e/jest.config.ts#L12

Reply:
Yes, they need to be present in dev dependecies because we are now using `yarn focus` instead of yarn install